scripts/controllers/ObstacleAvoidanceController.py

```diff
@@ -0,0 +1,348 @@
+#!/usr/bin/env python
+
+from CartesianPositionController import CartesianPositionController
+import numpy as np
+import rospy
+import tf
+
+NUMBER_OF_CONTROL_POINTS = 1
+
+# Parameters of the flacco paper
+MAX_RATE = 1.0  # Obstacle velocity estimation equation
+ALPHA = 6.0
+C = 5.0
+RHO = 0.4
+V_MAX = 0.21
+Q_DOT_MIN = np.array([-2.1750, -2.1750, -2.1750, -2.1750, -2.6100, -2.6100, -2.6100])
+Q_DOT_MAX = np.array([+2.1750, +2.1750, +2.1750, +2.1750, +2.6100, +2.6100, +2.6100])
```

Comment:
These are the same as those in the CartesianPositionController. Can we change this to be read in though params?